lib/serialize.js

```diff
@@ -56,78 +58,79 @@ export async function pageProps({ params }) {
     data.title = ''
   }
 
-  const importBasePath = `${process.cwd()}/content${slug
-    .split('/')
-    .slice(0, -1)
-    .join('/')}`
+  const mdxPathAbsolute = stdpath.join(process.cwd(), 'content', route.path)
 
-  const mdxSource = await serialize(content, {
-    parseFrontmatter: false,
-    scope: { data },
-    format: process.env.DOCS_USE_MDX === 'true' ? 'mdx' : 'md',
-    mdxOptions: {
-      rehypePlugins: [
-        rehypeSlug,
-        [
-          rehypeAutolink,
-          {
-            behavior: 'append',
-            properties: {
-              className: ['btn-copy-link', 'invisible']
-            },
-            content: {
-              type: 'element',
-              tagName: 'svg',
+  const mdxSource = await serialize(
+    new VFile({
+      value: content,
+      path: mdxPathAbsolute
+    }),
```

Comment:
Passing a `VFile` instead of just the MDX source, allows the plugins to know the path of the source file and calculate relative paths from it. This allows files to be included from anywhere beneath the repository root directory.